packages/design-system/src/components/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Button } from '../Button';
+import Alert from './Alert';
+
+const meta: Meta<typeof Alert> = {
```

Comment:
`title` no longer needs to be declared as Storybook can infer the component layout from the tree view in your project